app/precompiled.py

```diff
@@ -523,6 +425,34 @@ def _colour_no_print_areas(src_pdf, page_number=1):
         can.rect(page_width - right, 0, page_width, page_height, fill=True, stroke=False)
         # bottom margin:
         can.rect(left, 0, page_width - (2 * right), bottom, fill=True, stroke=False)
+
+        # The first page is more varied because of address blocks etc subsequent pages are more simple
+        if page_num == 0:
```

Comment:
In the deleted code `if page_num == 1` now it's 0, why is that?

Reply:
the deleted code was looking at a user provided page_num variable, which starts at page 1

this is defined in `for page_num in range(0, pdf.numPages)`, and so is 0 indexed